application/src/main/java/be/jpendel/domain/event/EventFactory.java

```diff
@@ -4,6 +4,7 @@
 
 public class EventFactory {
     //TODO add check that event with same name and same startdate, location can not be created
+    //->check EventFactoryTest
```

Comment:
Kunnen we over discussiëren. Valt voor beide iets te zeggen. Je wilt het sowieso ook op je db afchecken dan. want tussen het moment van creatie en persisteren kan er een nieuwe aangemaakt zijn.